docs/pages/admin-guides/access-controls/guides/dual-authz.mdx

```diff
@@ -199,19 +198,15 @@ Alice and Ivan can review and approve request using Web UI or CLI:
 
     # ID                                   User             Roles   Created (UTC)       Status
     # ------------------------------------ ---------------  ------- ------------------- -------
-    # 9c721e54-b049-4ef8-a7f6-c777aa066764 bob@example.com  dbadmin 03 Apr 21 03:58 UTC PENDING
+    # 0193496f-268c-727e-b696-600a868429ff test (Bob)  elevated-access 21 Nov 24 18:50 UTC PENDING
```

Comment:
Does this still look good in the rendered view? With the expanded role name it looks like the columns might not align anymore.

Reply:
Recreated the table, I think this should fix it

Reply:
Does this look ok:


![tsh_req_list_output](https://github.com/user-attachments/assets/b8f27272-31ea-4ab6-8ae0-135eef028083)